pages/typography.html

```diff
@@ -4,10 +4,30 @@
 title: Typography
 ---
 
-<p>The primary font is <strong>Helvetica Neue</strong>, in bold and regular weights.</p>
 
-<p>We use one typeface to keep things simple, and use variations in size and weight to create hierarchy.</p> 
+<div class="usa-grid-full">
+    <p>The primary font is <strong>Helvetica Neue</strong>, in bold and regular weights.</p>
+    <p>We use one typeface to keep things simple, and use different font sizes and weights to create hierarchy.</p> 
+    <p>Helvetica Neue is a system font on all Apple devices, so you don't need to download or install anything. On Windows devices, if Helvetica Neue is not available, use Arial as a replacement.</p>
+    <p>For help formatting text in Keynote, see <a href="https://support.apple.com/kb/PH16939?locale=en_US">Apple&rsquo;s guide to using paragraph styles</a>.</p>
 
-<p>Helvetica Neue is a system font on all Apple devices, so you don't need to download or install anything. On Windows devices, if Helvetica Neue is not available, use Arial as a replacement.</p>
+  <div class="typography-specimen usa-width-one-half usa-end-row">
+  <h4 class="usa-heading-alt">Helvetica Neue, Regular</h4>
```

Comment:
@thisisdano I'm _pretty sure_ we can drop the `class="usa-heading-alt"` and use a regular `h4` in this instance. We don't have any custom styles applied. That class is defined in the embedded USWDS here: https://github.com/18F/web-design-standards/blob/16e8619c9c5b39461d2645d6e8e7ca4db5d309fb/assets/css/styleguide.scss#L823